doc/api/api_changes/2017-12-20-JMK.rst

```diff
@@ -0,0 +1,20 @@
+`Figure.legend` no longer checks for repeated lines to ignore
```

Comment:
Can you put this directly in `api_changes.rst`.  The reason we put them in individual files is to avoid interminable rebases when the docs overlap, but this _should_ be the only 2.1.2 related API change so that is not a risk ;)

Reply:
No problem.  But I’ll be away from machine for a while.

Reply:
no worries!